python/metrics/src/hydrotools/metrics/metrics.py

```diff
@@ -195,9 +195,9 @@ def compute_contingency_table(
         
     Parameters
     ----------
-    observed: pandas.Series, required
+    observed: array-like, required
```

Comment:
why does this indicate an `array-like` type but the string below says `pandas.Series`?